Nos/Views/RepliesView.swift

```diff
@@ -102,17 +104,22 @@ struct RepliesView: View {
                             showReplyCount: false,
                             displayRootMessage: true,
                             isTapEnabled: false,
-                            replyAction: { _ in self.focusTextView = true },
+                            replyAction: { _ in self.showReplyComposer = true },
                             tapAction: { tappedEvent in tappedEvent.referencedNote().unwrap { router.push($0) } }
                         )
                         .padding(.top, 15)
+                        .sheet(isPresented: $showReplyComposer, content: {
+                            NewNoteView(replyTo: note, isPresented: $showReplyComposer)
+                                .environment(currentUser)
```

Comment:
I'm not sure, but I wonder if this is needed. Can you please test if it works without this line?

Reply:
This fixed an intermittent crash so I don't want to remove it, because I can't prove that removing it doesn't reintroduce the crash 🙃